### ISSUE TITLE
how to adjust the learning rate parameter of PyTorch-Multi-Label-Image-Classification code

### ISSUE THREAD

**Issue by liulan93** [spmallick/learnopencv#927]
As the code of train.py doesn't have learning rate parameter, how should I adjust this parameter?
https://github.com/spmallick/learnopencv/blob/master/PyTorch-Multi-Label-Image-Classification/train.py
